openshift/templates/bcfw_p1_hourly_actuals.cronjob.yaml

```diff
@@ -41,7 +50,7 @@ objects:
             spec:
               containers:
                 - name: ${JOB_NAME}
-                  image: docker-registry.default.svc:5000/${PROJECT_TOOLS}/${NAME}-${SUFFIX}:${SUFFIX}
+                  image: ${IMAGE_REGISTRY}/${PROJ_TOOLS}/${NAME}-${SUFFIX}:${SUFFIX}
```

Comment:
@andrea-williams have you re-run jenkins in prod on silver? It's failing there, still pulling from the old stuff.